config/config.yaml

```diff
@@ -7,7 +7,7 @@ extensions:
         endpoint: 0.0.0.0:55679
     ## Uncomment below if you want to enable authentication to GitHub
     ## Presume that GH_USER and GH_PAT are set as env variables.
-    #basicauth/gitlab:
+    #bearertokenauth/gitlab:
```

Comment:
this was supposed to be github, and gitlab is supposed to be commented out. also it's supposed to eb basicauth. if we want to change to bearer token (which actually is the better option) then we'll have to update the comments to remove the user information & change client auth.